ts-testing-training/src/0_numberStack/index.test.ts

```diff
@@ -1,5 +1,52 @@
-describe('Stack', () => {
-  it('should do stuff', () => {
-    // do stuff!
+import {NumberStack} from './index';
+
+describe('NumberStack', () => {
+  /* worked initially
```

Comment:
As a side note: If you want to initialize something for every test, you can use the `beforeEach` block like so:

```js
let testSubject: NumberStack;

beforeEach(() => {
  testSubject = new NumberStack();
})

  it('should return the last pushed value on pop() call', () => {
    testSubject.push(10);
    testSubject.push(5.5);

    expect(testSubject.pop()).toStrictEqual(5.5);
  });

// ...etc
```